app/models/bike.rb

```diff
@@ -8,8 +8,8 @@ class Bike < ApplicationRecord
     # end
 
 
-  CATEGORIES = ['City bike', 'Road bike', 'Mountain bike']
-  GEARS = ['3', '7', '18', '21', '28']
+  CATEGORIES = ['City bike', 'Road bike', 'BMX bike', 'Cargo bike', 'Mountain bike', 'Single-speed bike', 'Fixie bike']
+  GEARS = ['1', '3', '7', '18', '21', '28']
```

Comment:
First element of the array could be replaced by a string..

'No gears'

Reply:
done